openedx/adg/lms/applications/models.py

```diff
@@ -36,6 +45,18 @@ class ApplicationHub(TimeStampedModel):
         default=False, verbose_name=_('Written Application Submitted'),
     )
     submission_date = models.DateField(null=True, blank=True, verbose_name=_('Submission Date'), )
+    prerequisite_course_status = models.CharField(
+        verbose_name=_('Program Prerequisite Courses Status'),
+        choices=STATUS_CHOICES,
+        max_length=15,
+        default=NOT_STARTED,
+    )
+    bu_prerequisite_course_status = models.CharField(
```

Comment:
the field name should contain the plural **courses**

`bu_prereq_courses_status`